CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ include(${BASE_LOCATION}/toolchain/arm-gcc.cmake)
 # Configure project and languages
 project(efm32-test C CXX ASM)
 
+set(DEVICE EFM32TG11B120F128GQ48)
```

Comment:
it's great to have available options documented here, but am hesitant to provide a default that is less common than the one that was already there? i think the JLink options commented out / with a comment could be good.

Reply:
Yes.. actually that's leakage from my own project.. ;) Should be fixed now..